src/main/java/com/paneedah/mwc/entities/BanditEntityFactory.java

```diff
@@ -29,9 +29,10 @@ public void createEntity(ModContext modContext) {
         .withMaxHealth(40)
         .withEntityIdSupplier(() -> 10000)
         .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.EASY, 0.1f, Magazines.VSSVintorezMag)
+        .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.HARD, 0.1f, Magazines.ASValMag)
```

Comment:
What's the difference between the two

Reply:
ASValMag should have 30rnds instead of 20rnds